dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/FATSuite.java

```diff
@@ -41,14 +42,19 @@ public class FATSuite {
     public static final String jakartaeeServer = "com.ibm.ws.jca.fat.jakarta";
 
     /*
-     * EE7 will run with full fat only. EE9 will be run with lite and full fat.
+     * EE7 will run in full mode only.
+     * EE9 will run in full mode only.
+     * EE10 will run in full and lite mode.
      */
     @ClassRule
     public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly())
-                    .andWith(new JakartaEE9Action().withWiden()); // need widen option to handle jar file within a jar file.
+                    // need widen option to handle jar file within a jar file.
+                    .andWith(new JakartaEE9Action().withWiden())
+                    // need widen option to handle jar file within a jar file.
+                    .andWith(new JakartaEE10Action().withWiden());
```

Comment:
I think EE9 will run in Lite at the moment ... should it have `fullFATOnly()` on it?